nixos/modules/services/networking/bind.nix

```diff
@@ -125,14 +127,16 @@ in
             master=false means slave server; slaves means addresses
            who may request zone transfer.
         ";
-        example = [{
-          name = "example.com";
-          master = false;
-          file = "/var/dns/example.com";
-          masters = ["192.168.0.1"];
-          slaves = [];
-          extraConfig = "";
-        }];
+        example = [
```

Comment:
while you're at it - please use `literalExample` here for better formatting in the manual.

Reply:
fixed